### PR TITLE
fix on the check of the profile parsing and the scaling of the ATES

### PR DESCRIPTION
**Pull request by FJanssen-TNO**
Fix on the profile parsing to check, to allow for no heatingdemand power to be filled in at the ESDL.
Fix on the scaling of the ATES temperature changes, when the temperature modelling is not used.

Running the municipality example with HIGHS remains too computationally intensive to be included in the test_examples.py.

### CODE REVIEW

**Comment by FJanssen-TNO** [comment]
@KobusVanRooyen ready for review

File: src/mesido/esdl/profile_parser.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def read_profiles(
                     # Check if that the installed heat/cool demand capacity is sufficient
                     if component_type in ["heat_demand", "cold_demand"]:
                         max_profile_value = max(values)
-                        if asset_power < max_profile_value:
+                        if asset_power < max_profile_value and asset_power != 0.0:
```
Comment (KobusVanRooyen) [inline src/mesido/esdl/profile_parser.py:137]: Update the CHANGELOG

**Comment by KobusVanRooyen** [comment]
@FJanssen-TNO the review has been completed

**Comment by KobusVanRooyen** [comment]
@FJanssen-TNO approved

**Pull request merged**